shared/actions/config/index.js

```diff
@@ -534,6 +563,7 @@ function* configSaga(): Saga.SagaGenerator<any, any> {
   // Kick off platform specific stuff
   yield Saga.spawn(PlatformSpecific.platformConfigSaga)
   yield Saga.spawn(avatarSaga)
+  yield Saga.spawn(criticalOutOfDateCheck)
```

Comment:
No need to wait for `installerRan`? Is this guaranteed to run after service is up?

Reply:
Don't think it matters. Checks every hour

Reply:
It'd probably black bar the first time if it races. Also it's nice to not miss the first one.

Reply:
No black bar there's a try

Reply:
Ah you're right. OK